blivet/devices/storage.py

```diff
@@ -190,6 +190,14 @@ def raw_device(self):
         """ The device itself, or when encrypted, the backing device. """
         return self
 
+    @property
+    def controllable(self):
+        return self._controllable and not flags.testing and not self.unavailable_type_dependencies()
+
+    @controllable.setter
+    def controllable(self, value):
+        self._controllable = value
```

Comment:
There are a few places from the populator where we update the controllable property for some special cases.